### PR TITLE
Feature/serve frontend via proxy

### PR DESCRIPTION
**Pull request by connoratrug**
Proxy frontend
- put nginx in front of molgenis
- expose nginx on port 80
- proxy @molgenis-ui to unpackage
- proxy everything else to molgenis service

documentation: https://docs.google.com/document/d/1Lj-aHBBqYaAvtb0YPF9UqbNAZMcmeEtSpHinWlzdPq4/edit#

#### Checklist
- Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/integration/system tested
- Migration step added in case of breaking change
- User documentation updated
- (If you have changed REST API interface) view-swagger.ftl updated
- Test plan template updated
- Clean commits
- Added Feature/Fix to release notes

### CODE REVIEW

File: molgenis-app-manager/src/main/resources/templates/view-app-manager.ftl [hunk 1]
```diff
@@ -2,15 +2,17 @@
...
 <#assign js=[]>
 <#assign version = 2>
 <@header css js version/>
 
 <div id="app-manager"></div>
 
-<script type=text/javascript src="<@resource_href "/js/app-manager/manifest.js"/>"></script>
-<script type=text/javascript src="<@resource_href "/js/app-manager/vendor.js"/>"></script>
-<script type=text/javascript src="<@resource_href "/js/app-manager/app.js"/>"></script>
+<link rel="stylesheet" href="/@molgenis-ui/app-manager/dist/css/app-manager/app.css"/>
```
Comment (fdlk) [inline molgenis-app-manager/src/main/resources/templates/view-app-manager.ftl]: Let's remove the superfluous app name in the dist dir

**Comment by fdlk** [comment]
What do we do with the themes?

**Comment by connoratrug** [comment]
> What do we do with the themes?

for now leave them as is, and write story for new theme workflow, check with PM for feature input.

File: molgenis-app-manager/src/main/resources/templates/view-app-manager.ftl [hunk 2]
```diff
@@ -2,15 +2,17 @@
...
 <@header css js version/>
 
 <div id="app-manager"></div>
 
-<script type=text/javascript src="<@resource_href "/js/app-manager/manifest.js"/>"></script>
-<script type=text/javascript src="<@resource_href "/js/app-manager/vendor.js"/>"></script>
-<script type=text/javascript src="<@resource_href "/js/app-manager/app.js"/>"></script>
+<link rel="stylesheet" href="/@molgenis-ui/app-manager/dist/css/app.css"/>
+
+<script type=text/javascript src="<@resource_href "/@molgenis-ui/app-manager/dist/js/manifest.js"/>"></script>
```
Comment (fdlk) [inline molgenis-app-manager/src/main/resources/templates/view-app-manager.ftl]: @resource_href only works for files on the classpath

**Comment by fdlk** [comment]
> > What do we do with the themes?
> 
> for now leave them as is, and write story for new theme workflow, check with PM for feature input.
Didn't work because their location changes. I've re-added the themes to core-ui resources.

**Comment by fdlk** [comment]
> Metadata manager, search all, one-click importer, app manager, questionnaire and security ui are still missing fontawesome

Thanks for spotting! I've re-added the fonts to the classpath of core-ui.

**Pull request merged**